index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <h1 id="title">OUR SURVEY FORM</h1>
```

Comment:
Please use <header> in h1 for best practice of semantic html

Reply:
Great job being polite and giving a reason for the requested change.

Reply:
Great job being polite and giving a reason for the requested change.